sdk/confidentialledger/azure-security-confidentialledger/README.md

```diff
@@ -137,7 +110,7 @@ This project has adopted the [Microsoft Open Source Code of Conduct][coc]. For m
 [ccf]: https://github.com/Microsoft/CCF
 [azure_confidential_computing]: https://azure.microsoft.com/solutions/confidential-compute
 [confidential_ledger_docs]: https://aka.ms/confidentialledger-servicedocs
-[samples]: src/samples/java/com/azure/security/confidentialledger
+[samples]: https://github.com/Azure/azure-sdk-for-java/tree/main/sdk/confidentialledger/azure-security-confidentialledger/src/samples/java/com/azure/security/confidentialledger/generated
```

Comment:
you should write your own sample.

generated ones are useful to get you started, but it might not be the correct one for customer.